client/src/components/landing/Landing.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import { authActions } from '../../sagas/actions';
+
+const Landing = ({ register }) => {
+  const [formData, setFormData] = useState({ id: '', name: '' });
```

Comment:
I imagine we're going to continue to add more state; do we need to care about "default values" or do you think that this is fine since it's the landing page?

Reply:
For the landing page, you'd want empty defaults (which corresponds to empty form inputs). I haven't tried, but if you put a default here it _should_ prepopulate those inputs (which we don't want)

Reply:
Remember, this state is local to `Landing`